Assignment3/app.py

```diff
@@ -50,9 +50,9 @@ def __init__(self, master=None):
         self.receive_thread = Thread(target=self._ReceiveMessages, daemon=True)
 
         # Creating a protocol object
-        self.prtcl = Protocol("initSharedKey")
+        self.prtcl = Protocol(None)
         # Note the Updated GUI key is caputred upon trying to connect
-        self._CaptureSharedSecret()
+    
 
```

Comment:
I wrote notes to help speed up the change process for the tests. Viewable here. For reference for either you or me if I get to it first thursday

https://github.com/cjohst/CPEN442-assignments/commit/c080db14786f5b9187cee7be59ce4c9847b0ba59#diff-24fe528b8b1cffe4461ea0668f6362f4b5367a2ee2c656d4c44620ac3205ad59